kuma/core/tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+from datetime import date, timedelta
+
+from nose.tools import eq_
+
+from django.test import TestCase
+
+from kuma.core.models import IPBan
+
+
+class RevisionIPTests(TestCase):
+    def test_delete_older_than_default_30_days(self):
+        old_date = date.today() - timedelta(days=31)
+        IPBan.objects.create(ip='127.0.0.1', created=old_date).save()
+        eq_(1, IPBan.objects.all().count())
```

Comment:
`all()` is not needed here, it's implied when running `count()`, also happens in lines below